plugins/Jobs.cpp

```diff
@@ -787,16 +787,29 @@ void BedrockJobsCommand::process(SQLite& db) {
         if (!retriableJobs.empty()) {
             for (auto job : retriableJobs) {
                 SINFO("Updating job with retryAfter " << job["jobID"]);
+                STable jobData = SParseJSONObject(job["data"]);
+                if (SToInt(jobData["retryAfterCount"]) >= 10) {
```

Comment:
I was wondering if we needed to check if `jobData["retryAfterCount"]` exists but looking at what `SToInt` does: https://github.com/Expensify/Bedrock/blob/9b8fe5493e3ae6787830a65c6930cc631ec9164b/libstuff/libstuff.cpp#L2739-L2741

we're fine right? This is because the `val.c_str()` will return a c string that is null terminated (`\0`) https://www.cplusplus.com/reference/string/string/c_str/ and `atoi` will return 0 if the passed c string has only whitespace characters or a non valid integer value. 

so if `jobData["retryAfterCount"]` isn't set for a job then the c string returned by `val.c_string()` will just be the null terminated value and `atoi` will return 0 for that. 

Just rubber ducking this out to make sure I'm following it properly.